argocd/applications/configs/cluster-manager.yaml

```
@@ -5,3 +5,13 @@
 clusterManager:
   extraArgs:
     disable-mt: false
+
+templateController:
+  metrics:
+    service:
+      enabled: true
+      port: 8080
+      labels:
+        app: "template-controller-metrics-svc"
+    serviceMonitor:
+      enabled: true
```

```suggestion
templateController:
  metrics:
    service:
      enabled: true
    serviceMonitor:
      enabled: true
```

do we need to redefine port and labels once again?

---

good point. Removed anything that's already set by default